examples/resources/configuration/certificates.yaml

```
@@ -25,70 +24,64 @@ data:
           "widget": "Panel",
           "children": [
             {
-              "path": "spec.commonName",
-              "type": "string",
-              "name": "commonName"
+              "source": "spec.commonName",
+              "name": "Common name"
             },
             {
-              "path": "spec.duration",
-              "type": "string",
-              "name": "duration"
+              "source": "spec.duration",
+              "name": "Duration"
             },
             {
-              "path": "spec.encodeUsagesInRequest",
-              "type": "boolean",
+              "source": "spec.encodeUsagesInRequest",
               "name": "encodeUsagesInRequest"
             },
             {
-              "path": "spec.isCA",
-              "type": "boolean",
-              "name": "isCA"
+              "source": "spec.isCA",
+              "name": "Is CA"
             },
             {
-              "path": "spec.renewBefore",
+              "source": "spec.renewBefore",
               "type": "string",
-              "name": "renewBefore"
+              "name": "Renew before"
             },
             {
-              "path": "spec.revisionHistoryLimit",
-              "type": "integer",
-              "name": "revisionHistoryLimit"
+              "source": "spec.revisionHistoryLimit",
+              "name": "Revision History Limit"
             },
             {
-              "path": "spec.secretName",
-              "type": "string",
+              "source": "spec.secretName",
               "name": "secretName"
```

```suggestion
              "name": "Secret name"
```
since we're adjusting everything else...